File: JavaScript/Code/floyd_triangle.js
const floyd = (max_row) => {
    let column_count = 1;
    let num = 1;
    for (i = 0; i < max_row; i++) {
        row = '';
        for (j = 0; j < column_count; j++) {
            row += ' ' + num;
            num++;
        }
        console.log(row);
        column_count++;
    }
}

floyd(5);

File: Python/floyd_triangle.py
def main(max_row):
    column_count = 1
    num = 1
    for row in range(max_row):
        for column in range(column_count):
            print(num, end=" ")
            num += 1
        print()
        column_count += 1



if __name__ == '__main__':
    num = int(input("Enter now of rows : "))
    main(num)